src/test/parameters.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import nock from 'nock';
-import {ShopperCustomers} from '../lib/shopperCustomers';
+import {ShopperSearch} from '../lib/shopperSearch';
```

Comment:
Did we do this change as the actions/login is deprecated?

Reply:
Yes that is correct